code/datums/Vspace.dm

```diff
@@ -212,6 +212,11 @@ datum/v_space
 			virtual_character.real_name = "Virtual [user.real_name]"
 		user.mind.virtual = virtual_character
 		user.mind.transfer_to(virtual_character)
+		// We do this here because transfer_to overwrites the say_tree
+		if (!virtual_character.isghost)
+			virtual_character.ensure_say_tree().AddOutput(SPEECH_OUTPUT_SPOKEN)
+			virtual_character.ensure_say_tree().RemoveOutput(SPEECH_OUTPUT_DEADCHAT)
+			virtual_character.default_speech_output_channel = SAY_CHANNEL_OUTLOUD
```

Comment:
Could this be done on the virtual human's `New()`, or does that break setup?

Reply:
That was my first attempt -- however, this proc seems to overwrite the say tree with the tree of the mob joining Vspace. Either the channels would need to be added here as-is, or transferring into a client would need to not overwrite the mob's say tree input/outputs.

TL;DR breaks setup

Reply:
That's odd; I don't think transferring clients transfers mob say tree outputs? It only transfers OOC modules and similar.

Reply:
I stepped through this proc, in `user.mind.transfer_to(virtual_character)` it was happening